plugins/file/index.js

```diff
@@ -13,13 +13,16 @@ function getDateString() {
   return `${now.getFullYear()}/${month}/${day}`;
 }
 
-async function upload(s3, params) {
+async function upload(s3Client, params, s3Region) {
   return new Promise((resolve, reject) => {
-    s3.upload(params, (err, data) => {
+    s3Client.send(new PutObjectCommand(params), (err) => {
```

Comment:
I just saw that the new aws-sdk library now supports async/await. It would be great to refactor this code to use async/await. See the documentation for more details: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-s3/index.html#asyncawait